packages/blade/src/utils/useUpdateEffect.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Add tests

Reply:
Add tests and comments in subsequent PR if needed since its a P0

Reply:
added, just realized most of our hooks don't have tests, we should add them.